solnlib/log.py

```diff
@@ -251,6 +252,31 @@ def modular_input_end(logger: logging.Logger, modular_input_name: str):
     )
 
 
+def _base_error_log(
+    logger,
+    exc: Exception,
+    exe_label,
+    full_msg: bool = True,
+    msg_before: str = None,
```

Comment:
As `msg_before` and `msg_after` are str, can we keep the default value as "" (blank string)?

Reply:
hmm we can but this value is only used to check whether this variable should be used. We do not pass the default value anywhere